### ISSUE TITLE
[Question] Sniff to ensure `@var` phpDoc order

### ISSUE THREAD

**Issue by leon0399** [slevomat/coding-standard#1576]
# Example:

```php
/** @var string $correct */
$correct = $foo->bar();

/** @var $incorrect string */
$incorrect = $foo->bar();
```

Notice the docblock params order
